{{cookiecutter.project_name}}/contracts/NFT.vy

```diff
@@ -133,16 +127,23 @@ EIP712_DOMAIN_VERSIONHASH: constant(bytes32) = keccak256("1")
 NAME: constant(String[20]) = "{{cookiecutter.token_name}}"
 SYMBOL: constant(String[5]) = "{{cookiecutter.token_symbol}}"
 IDENTITY_PRECOMPILE: constant(address) = 0x0000000000000000000000000000000000000004
+{%- if cookiecutter.updatable_uri == 'y' %}
+baseURI: public(String[100])
```

Comment:
`baseURI` is only available as a public method if cookiecutter.updatable_uri is set